burnman/eos/mie_grueneisen_debye.py

```diff
@@ -174,6 +174,10 @@ def validate_parameters(self, params):
         """
         Check for existence and validity of the parameters
         """
+        if 'T_0' not in params:
+            params['T_0'] = 300.
+        if 'P_0' not in params:
+            params['P_0'] = 0.
```

Comment:
Is this used anywhere in this EoS? My recollection is that all the pressure stuff is turfed off to the isothermal Birch-Murnaghan.